### ISSUE TITLE
Problem customizing bowtie2 options

### ISSUE THREAD

**Issue by vr1087** [barricklab/breseq#159]
I am running: 
```
breseq 0.32.0
---> bowtie2  :: version 2.3.4.1 [/usr/local/bin/bowtie2]
---> R        :: version 3.0.2 [/usr/bin/R]
```
I am trying to use custom bowtie2 options with this command:

```
breseq --num-processors 30 --output negatives --polymorphism-prediction -r ecoli.gb \
--bowtie2-scoring "--ignore-quals --mp 2 --np 0 --rdg 2,3 --rfg 2,3" \
--bowtie2-stage1 "--end-to-end -i S,1,0.25 --score-min G,-295,-0.9" \
--bowtie2-stage2 "" \
--bowtie2-junction "--end-to-end -i S,1,0.25 --score-min G,-295,-0.9" \
--polymorphism-frequency-cutoff 0.01 \
--brief-html-output /tmp/tmpTrKNF2.fastq /tmp/tmp70Ra5T.fastq
```
However, looking at breseq's stdout, it seems that the bowtie2-scoring argument is not being passed to bowtie 2's command line. Instead it seems that bowtie2-stage1 is passed twice:

```
[system] bowtie2 -t -p 30 -L 31 --end-to-end -i S,1,0.25 --score-min G,-295,-0.9 --end-to-end -i S,1,0.25 --score-min G,-295,-0.9 --reorder -x negatives/02_reference_alignment/reference -U negatives/01_sequence_conversion/tmpTrKNF2.converted.fastq -S negatives/02_reference_alignment/tmpTrKNF2.reference.sam

[system] bowtie2 -t -p 30 -L 31 --end-to-end -i S,1,0.25 --score-min G,-295,-0.9 --end-to-end -i S,1,0.25 --score-min G,-295,-0.9 --reorder -x negatives/02_reference_alignment/reference -U negatives/01_sequence_conversion/tmp70Ra5T.converted.fastq -S negatives/02_reference_alignment/tmp70Ra5T.reference.sam
```
I double checked that these bowtie2 arguments are reported in the SAM headers. Seems like an easy bug to fix.

**Comment by jeffreybarrick** [comment]
If you set the `--bowtie2-stage2` option to blank (""), then _breseq_ completely skips the 2nd alignment phase. So, your output is showing it do the first (and now only) alignment step on each of the two read files one after another. Is this the behavior you are aiming for? If you'd like it to use the normal options for 1st stage alignment instead, then omit the `--bowtie2-stage2` option.

**Comment by vr1087** [comment]
Yes, that is the behavior I was looking for. However, I was expecting the `--bowtie2-scoring` string to be passed to bowtie 2's command line, but it was not. Instead, I see the `--bowtie2-stage1` string passed _twice_ to bowtie 2's command line. Does that make sense? I tried omitting `--bowtie2-stage2` entirely, but I still observed this behavior.

**Comment by jeffreybarrick** [comment]
For your two input reference files tmpTrKNF2.fastq and tmp70Ra5T.fastq there would normally be four bowtie2 commands processed (stage1 and stage 2 for read file 1 followed by stage1 and stage2 for read file2). The output you pasted shows two bowtie2 commands which are both proper and expected stage1 commands (one for each input file) and no stage2 commands, one per input file. The commands are different, so it seems to be operating properly to me?

**Comment by vr1087** [comment]
Right, I understood that. The problem is that my custom alignment scoring parameters are not being passed to bowtie 2.

**Comment by vr1087** [comment]
Excellent. Thank you Jeff.
